lib/services/network_info_service.dart

```diff
@@ -20,29 +20,30 @@ class NetworkInfoService with LogMixin, ChangeCallbackProvider {
   UnmodifiableListView<String> get ips => UnmodifiableListView(_ips);
 
   Future<String?> get ssid {
-    l.info('request for getting ssid');
+    logger.info('request for getting ssid');
     if (_ssidCompleter != null && !_ssidCompleter!.isCompleted) {
-      l.warning('waiting for previous ssid request to finish');
+      logger.warning('waiting for previous ssid request to finish');
 
       return _ssidCompleter!.future.then((value) {
-        l.info('completer finished with ssid: "$value"');
+        logger.info('completer finished with ssid: "$value"');
 
         return value;
       });
     }
     _ssidCompleter = Completer();
 
     return _detectSsid().then((value) {
-      l.info('detected ssid: "$value"');
+      logger.info('detected ssid: "$value"');
       _ssidCompleter?.complete(value);
 
       return value;
     });
   }
 
   Future<String?> _detectSsid() async {
-    l.info('detecting ssid');
-    if (kIsWeb || !(Platform.isAndroid || Platform.isIOS)) return null;
+    logger.info('detecting ssid');
+    final isMobileDevice = Platform.isAndroid || Platform.isIOS;
```

Comment:
This line throws in the browser version because the browser doesn't have access to the Platform class (thus the network section in the settings screen just shows the text error). That's the reason why I put it after the `kIsWeb` check. A working alternative would be to make isMobileDevice a function that returns the result.